.travis.yml

```diff
@@ -1,9 +1,13 @@
 language: node_js
 node_js: ["0.10"]
+cache:
+  directories:
+    - node_modules
+    - bower
 before_install:
   - export CHROME_BIN=chromium-browser
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 before_script:
-  - npm install -g bower gulp
```

Comment:
Oh, I see. `npm-test` will choose to use local scripts before considering global scripts. [[READ MORE](https://docs.npmjs.com/cli/run-script)].

If you want to test, I would rerun the build without cache dependencies. Locally, you can issue the following commands:
```
$ npm uninstall -g gulp
$ gulp  # bash: command not found: gulp
$ npm test   # observe that gulp works
```